ui/components/app/simulation-preview/simulation-preview.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { Provider } from 'react-redux';
+import configureStore from '../../../store/store';
+import { SimulationPreview } from './simulation-preview';
+import mockState from '../../../../test/data/mock-state.json';
+
+const storeMock = configureStore({
+  metamask: {
+    ...mockState.metamask,
+    preferences: {
+      ...mockState.metamask.preferences,
+      useNativeCurrencyAsPrimaryCurrency: false,
+    },
+  },
+});
+
+const meta: Meta<typeof SimulationPreview> = {
+  title: 'Components/App/SimulationPreview',
+  component: SimulationPreview,
+  decorators: [(story) => <Provider store={storeMock}>{story()}</Provider>],
+};
+
+export default meta;
+type Story = StoryObj<typeof SimulationPreview>;
+
+export const PositiveNativeChange: Story = {
+  args: {
```

Comment:
Do we also need to provide the `argTypes` so these values can be configured within storybook by the user?

Reply:
It sees to be configurable already. I can't think of a better input type for simulation data.